elide-core/src/main/java/com/yahoo/elide/core/EntityDictionary.java

```diff
@@ -55,6 +55,7 @@
     private final ConcurrentHashMap<String, Class<?>> bindJsonApiToEntity = new ConcurrentHashMap<>();
     private final ConcurrentHashMap<Class<?>, String> bindEntityToJsonApi = new ConcurrentHashMap<>();
     private final ConcurrentHashMap<Class<?>, AccessibleObject> bindIdField = new ConcurrentHashMap<>();
+    private final ConcurrentHashMap<Class<?>, Class<?>> bindIdType = new ConcurrentHashMap<Class<?>, Class<?>>();
```

Comment:
To stay clean, you can probably just use the diamond operator here like above (i.e. `<>` instead of specifying type)